### DISCUSSION TITLE
SOF v1.6 release

### DISCUSSION THREAD

**Discussion by dbaluta** [thesofproject/sof#7385] [Team Posts]
Cc: @lgirdwood @plbossart @lbetlej 

**ETA** : TBD

Features:
- post process support
- support for NXP i.MX8MP
- initial support for NXp i.MX8ULP
- fuzzer for i.MX8

**Comment by lgirdwood** [comment]
@dbaluta rc3 now released but wont be final rc. Fixes still being merged into stable branch. What NXP features need added to release notes ?

**Comment by dbaluta** [comment]
@lgirdwood:

- support for NXP i.MX8MP
- Fuzzer host support for i.MX8/i.MX8X 
- various fixes for i.MX drivers

**Comment by lgirdwood** [comment]
@dbaluta thanks - anything pending - I think we have another week or 2 prior to v1.6 final ?

**Comment by cujomalainey** [comment]
@dbaluta where and how are you fuzzing?

**Comment by dbaluta** [comment]
@cujomalainey See ``sof/tools/fuzzer/platform/imx8-host.c``

We are able to parse topologies and send the IPCs via Qemu bridge to Qemu DSP instance.

Basically that was already done, we just added IMX host support in fuzzer and IMX DSP support in qemu.

**Comment by cujomalainey** [comment]
ah this was using the existing infra, got it.
